### ISSUE TITLE
Mock API fro development

### ISSUE THREAD

**Issue by kristenmills** [Flexget/webui#36]
I think at some point we should make a mock API for development. I usually just work off a snapshot of my actual flexget db but that's really not all that convenient

**Comment by shadycuz** [comment]
How helpful is this? I point at my own flexget 👀 . Something like this? https://hackernoon.com/liberate-your-ui-development-setup-a-mock-api-9b6167d06ffa

**Comment by kristenmills** [comment]
Yeah i haven't decided whether the benefits of doing it are worth yet since i don't personally need it.
